syncto/authentication.py

```diff
@@ -38,5 +40,18 @@ def build_sync_client(request):
 
     bid_assertion = authorization_header.split(" ", 1)[1]
     client_state = request.headers[CLIENT_STATE_HEADER]
-    sync_client = SyncClient(bid_assertion, client_state)
+    try:
+        sync_client = SyncClient(bid_assertion, client_state)
+    except HTTPError as e:
+        if e.response.status_code in (400, 401):
```

Comment:
Well SyncServer can return a 403 but the TokenServer cannot in this setup phase we are calling the tokenserver.
https://github.com/mozilla-services/syncclient/blob/master/sync/client.py#L54